package.json

```diff
@@ -9,10 +9,21 @@
     "url": "git@github.com:shramov/leaflet-plugins.git"
   },
   "scripts": {
-    "test": "eslint --config .eslintrc control/* layer/*"
+    "test": "./node_modules/.bin/karma start && npm run lint",
```

Comment:
Would a compromise be to just run lint here? I can add another script call for "specs" . That way you don't "have" to run it every-time.